src/applications/vaos/referral-appointments/components/ReferralTaskCardWithReferral.jsx

```diff
@@ -19,9 +19,7 @@ export default function ReferralTaskCardWithReferral() {
 
   const params = new URLSearchParams(search);
   const id = params.get('id');
-  const { data: referral, error, isLoading } = useGetReferralByIdQuery(id, {
-    skip: !id,
-  });
+  const { data: referral, error, isLoading } = useGetReferralByIdQuery(id);
```

Comment:
This isn't being used yet but if it was I think we would need to skip when there's no id in the url.